src/scenes/AcceleratorRouter/Applications/ApplicationView.tsx

```diff
@@ -1,10 +1,233 @@
 
 const ApplicationView = () => {
-  return <Page title={strings.APPLICATION} contentStyle={{ display: 'block' }}></Page>;
+  const { activeLocale } = useLocalization();
+  const theme = useTheme();
+  const { selectedApplication, setSelectedApplication, applicationSections, applicationDeliverables } =
+    useApplicationData();
+  const pathParams = useParams<{ applicationId: string }>();
+
+  useEffect(() => {
+    setSelectedApplication(Number(pathParams.applicationId ?? -1));
+  }, [setSelectedApplication, pathParams]);
+
+  const crumbs: Crumb[] = useMemo(
+    () => [
+      {
+        name: activeLocale ? strings.APPLICATION_LIST : '',
+        to: `${APP_PATHS.ACCELERATOR_APPLICATIONS}`,
+      },
+    ],
+    [activeLocale]
+  );
+
+  const titleComponent = useMemo(() => {
+    if (!selectedApplication || !activeLocale) {
+      return undefined;
+    }
+
+    return (
+      <TitleBar
+        header={strings.formatString(strings.DELIVERABLE_PROJECT, selectedApplication.projectName ?? '').toString()}
+        title={selectedApplication.internalName}
+      />
+    );
+  }, [selectedApplication]);
+
+  const prescreenSection = useMemo(
+    () => applicationSections.find((section) => section.phase === 'Pre-Screen'),
+    [applicationSections]
+  );
+
+  const nonPrescreenSections = useMemo(
+    () => applicationSections.filter((section) => section.phase === 'Application'),
+    [applicationSections]
+  );
+
+  const sectionDeliverables = useCallback(
+    (sectionId: number) => {
+      return applicationDeliverables.filter((deliverable) => deliverable.moduleId === sectionId);
+    },
+    [applicationDeliverables]
+  );
+
+  if (!selectedApplication || !prescreenSection || !nonPrescreenSections) {
+    return undefined;
+  }
+
+  return (
+    <Page crumbs={crumbs} title={titleComponent} contentStyle={{ display: 'block' }}>
+      <Card
+        style={{
+          display: 'flex',
+          flexDirection: 'column',
+          flexGrow: 1,
+          alignItems: 'center',
+          padding: theme.spacing(3),
+        }}
+      >
+        <Box display={'flex'} flexDirection={'column'} justifyContent={'left'} width={'100%'}>
+          <ApplicationReview application={selectedApplication} />
+
+          <Typography fontSize={'24px'} fontWeight={600} lineHeight={'32px'} marginTop={theme.spacing(3)}>
+            {strings.PRESCREEN}
+          </Typography>
+
+          {/* Add link to view boundary once exists */}
+          <ApplicationDeliverableRow title={strings.PROPOSED_PROJECT_BOUNDARY} goToDeliverable={() => {}} />
+
+          {sectionDeliverables(prescreenSection.moduleId).map((deliverable, index) => (
+            // Add link to deliverable
+            <ApplicationDeliverableRow
+              title={deliverable.name}
+              modifiedDate={
+                deliverable.modifiedTime ? DateTime.fromISO(deliverable.modifiedTime).toFormat('yyyy/MM/dd') : undefined
+              }
+              goToDeliverable={() => {}}
```

Comment:
Is this intentional? Or maybe handled in a future PR?

Reply:
Yes we don't have deliverables route yet!